<!-- -->
<launch>
    <arg name='respect_traffic_rules' default='True'/>
     <arg name='always_u_turn' default='False'/>

  <node pkg="psaf_global_planner" type="path_supervisor_common_roads.py" name="path_provider" output="screen">
      <param name='respect_traffic_rules' value='$(arg respect_traffic_rules)'/>
  </node>

    <node pkg="psaf_global_planner" type="planning_preprocessor.py" name="planning_preprocessor" output="screen">
      <param name='respect_traffic_rules' value='$(arg respect_traffic_rules)'/>
      <param name='always_u_turn' value='$(arg always_u_turn)'/>
  </node>
</launch>